plaso/cli/extraction_tool.py

```diff
@@ -121,7 +121,16 @@ def _CreateProcessingConfiguration(self, knowledge_base):
           'Unable to expand presets in parser filter expression with '
           'error: {0!s}').format(exception))
 
-    # TODO: validate parser names.
+    _, invalid_parser_elements = (
+        parsers_manager.ParsersManager.CheckFilterExpression(
+            parser_filter_expression))
+
+    if invalid_parser_elements:
+      invalid_parser_names_string = ','.join(invalid_parser_elements)
+      raise errors.BadConfigOption(
+          'Unknown element(s): "{0:s}" in parser filter '
```

Comment:
Maybe "Unsupported names of parsers or plugins found in element(s): "{0:s}" of parser filter."

Reply:
Done